package.json

```diff
@@ -41,6 +41,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.3.4",
+    "@emotion/core": "^10.0.10",
```

Comment:
What is this?

Reply:
I do not know why it stop working without it but Storybook requires it to run. This is odd, this happens after branch was rebased with master.

Reply:
Fixed in latest alpha (https://github.com/storybooks/storybook/issues/5817) so I will upgrade it.